server/src/main/java/io/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -444,13 +445,11 @@ public void onComplete(Result result)
                         TimeUnit.NANOSECONDS.toMillis(requestTimeNs),
                         "success",
                         success
-                        && result.getResponse().getStatus() == javax.ws.rs.core.Response.Status.OK.getStatusCode()
+                        && result.getResponse().getStatus() == Status.OK.getStatusCode()
```

Comment:
are these changes part of this PR?

Reply:
Well, this is a simple change to make code look better.